### DISCUSSION TITLE
stat_client卸载咨询

### DISCUSSION THREAD

**Discussion by joyoner** [zdz/ServerStatus-Rust#166] [Q&A] [answered]
参考以下doc使用一键命令安装非常便捷，请问有没有办法通过此类命令卸载呢？
https://doc.ssr.rs/rapid_deploy/

# stat_client 一键安装命令
curl -sSLf "https://ssr.rs/i?pass=p1&uid=h1" | bash

**Comment by zdz** [comment, marked as answer]
无，参考 https://github.com/zdz/ServerStatus-Rust/issues/152

**Reply by joyoner** [reply]
我摸索了下，貌似下面命令可行，但新引入了一个脚本

wget --no-check-certificate -qO status.sh 'https://raw.githubusercontent.com/zdz/ServerStatus-Rust/master/status.sh'
bash status.sh -u -c

**Reply by zdz** [reply]
停止服务，把目录删掉就行了，何必搞到这么麻烦？

**Reply by joyoner** [reply]
主要是有残留、因为服务虽然停掉了，但还在
